l1-contracts/test/foundry/unit/concrete/AddressAliasHelper/_AddressAliasHelper_Shared.t.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {AddressAliasHelperTest} from "solpp/dev-contracts/test/AddressAliasHelperTest.sol";
+
+contract AddressAliasHelperSharedTest is Test {
```

Comment:
Please add `test()` function to remove this file from the coverage?

Reply:
done